lambdas/service/app.py

```diff
@@ -746,7 +746,7 @@ def validate_filters(filters):
     Traceback (most recent call last):
     ...
     chalice.app.BadRequestError: BadRequestError: \
-    The `filters` parameter entry for `sampleDisease` must be a single-item dictionary.
+    The `filters` parameter entry for `sampleDisease` must be a dictionary.
```

Comment:
Don't really see the use case for allowing multiple relations. In particular, combining `is` with any of the others seems pointless. There may be a case for allowing a combination of the range-related relations.

Reply:
There's no intended explicit use case, but this solves the issue of the implicit source filter (`{"sourceId": {"is": [...]}}`) colliding with any of the range-related filters. A potential alternative approach would be to disallow those filters on `sourceId` by raising a `BadRequestError`. I presumed that this would be less disruptive.

Reply:
Decided at parking lot the alternative approach is preferable.